src/pages/edit-todo/index.tsx

```diff
@@ -1,7 +1,82 @@
-import InputField from "../../components/UI/InputField";
+import {
+  Box,
+  Button,
+  Container,
+  Grid,
+  Paper,
+  Stack,
+  TextField,
+} from "@mui/material";
+import { useSnackbar } from "material-ui-snackbar-provider";
+import { useForm } from "react-hook-form";
+import { useMutation } from "react-query";
 
-const EditTodo: () => JSX.Element = () => {
-  return <InputField />;
+import { ApiTodo } from "../../models/todo";
+import { editTodo } from "../../services/queries/edit-todo";
+import withAuthRedirect from "../../hoc/auth-redirect";
+
+const UpdateTodo = () => {
+  const { mutate } = useMutation(editTodo);
+
+  const snackbar = useSnackbar();
+
+  const { register, handleSubmit } = useForm({
+    defaultValues: { id: null, title: "", text: "", token: "" },
+  });
+
+  const submitHandler = (data: ApiTodo) => {
+    mutate(data);
```

Comment:
Shouldn't all these mutate calls use `await`

Reply:
Didn't see it used in the docs

Reply:
Leave it like it is currently